components/ILIAS/RTE/classes/class.ilRTE.php

```diff
@@ -166,6 +166,53 @@ public static function _cleanupMediaObjectUsage(string $a_text, string $a_usage_
         }
     }
 
+    /**
+     * Replace the latex delimiters used by the rich text editor
+     * Unfortunately these can't be processed by MathJax:
+     * "Note that the delimiters can’t look like HTML tags (i.e., can’t include the less-than sign),
+     * as these would be turned into tags by the browser before MathJax has the chance to run.
+     * You can only include text, not tags, as your math delimiters."
+     * @see https://docs.mathjax.org/en/latest/options/input/tex.html#option-descriptions
+     */
+    public static function _replaceLatexSpan(string $content): string
```

Comment:
IMO we could remove the leading `_` here. AFAIK this was a convention back in the day where the language did not support `static`.

```suggestion
    public static function replaceLatexSpan(string $content): string
```